subsys/settings/Kconfig

```diff
@@ -104,3 +110,18 @@ config SETTINGS_FS_MAX_LINES
 	depends on SETTINGS && SETTINGS_FS
 	help
 	  Limit how many items stored in a file before compressing
+
+config SETTINGS_NVS_SECTOR_SIZE_MULT
+	int "Sector size of the NVS settings area"
+	default 1
+	depends on SETTINGS && SETTINGS_NVS
+	help
+	  The sector size to use for the NVS settings area as a multiple of
+	  FLASH_ERASE_BLOCK_SIZE.
+
+config SETTINGS_NVS_SECTOR_COUNT
+	int "Sector count of the NVS settings area"
+	default 8
+	depends on SETTINGS && SETTINGS_NVS
+	help
+	  Number of sectors used for the NVS settings area
```

Comment:
Something wrong with new line, perhaps?
@kapi-no

Reply:
Do you want me to add a blank line at the end of this file?

Reply:
Yes, I think we should have blank lines at the end of all source files

Reply:
So we don't get these GH error signs.

Reply:
okay, done